### DISCUSSION TITLE
More Templates for 5.1.2

### DISCUSSION THREAD

**Discussion by leigh-pointer** [oqtane/oqtane.framework#4343] [Show and tell]
![image](https://github.com/oqtane/oqtane.framework/assets/9447612/a4720729-a6eb-46f1-9faa-9985d29f5835)

**Comment by leigh-pointer** [comment]
New addition
![image](https://github.com/oqtane/oqtane.framework/assets/9447612/8f9b3be9-af55-45e4-8360-14a0a7e5ce02)

**Comment by rchibnik** [comment]
Would you consider doing a DevExpress template?

**Reply by leigh-pointer** [reply]
I am busy with that but the component is very intense. Plus i need to get a trial license to integrate. Just because you have asked i will focus on it.

**Comment by rchibnik** [comment]
Thanks. If you need help testing it I have a license. Previously (v3) I had a template. Not as good as the code behind templates but it worked okI am willing to take it on perhaps using the syncfusion if you need helpCurrently I’m using radzen for my hobby project but we were using oqtane for lob applications it it is on hold for now. Our business apps use devx 
    Get Outlook for iOS
  

 From: Leigh Pointer ***@***.***>Sent: Wednesday, July 10, 2024 4:03 PMTo: oqtane/oqtane.framework ***@***.***>Cc: rchibnik ***@***.***>; Comment ***@***.***>Subject: Re: [oqtane/oqtane.framework] More Templates for 5.1.2 (Discussion #4343) 
I am busy with that but the component is very intense. Plus i need to get a trial license to integrate. Just because you have asked i will focus on it.
